packages/gatsby-core-utils/src/physical-cpu-count.ts

```diff
@@ -52,4 +52,4 @@ function getPhysicalCpuCount() {
   return fallbackToNodeJSCheck()
 }
 
-module.exports = getPhysicalCpuCount()
+export default getPhysicalCpuCount()
```

Comment:
Let's use Guidelines (from https://github.com/gatsbyjs/gatsby/issues/21995 description):
> Only use named exports, no export default.
```suggestion
export const physicalCpuCount = getPhysicalCpuCount()
```

and adjust all `physical-cpu-count` imports?

That should be in https://github.com/gatsbyjs/gatsby/blob/561d33e2e491d3971cb2a404eec9705a5a493602/packages/gatsby-core-utils/src/cpu-core-count.js#L6

and various tests